ui/learn/src/chessground.ts

```diff
@@ -18,6 +18,7 @@ export default function (ctrl: RunCtrl): VNode {
     hook: {
       insert: vnode => {
         const el = vnode.elm as HTMLElement;
+        el.addEventListener('contextmenu', e => e.preventDefault());
```

Comment:
We shouldn't need this, since we have `disableContextMenu: true,`, but that config setting alone wasn't working 🤔